### DISCUSSION TITLE
Cuando te emocionas con CSS en la practica de HTML

### DISCUSSION THREAD

**Discussion by carlos-gmx3** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#292] [Show and tell]
Como trabajé en una pastelería, aproveché y pedí permiso de usar los logos, de paso realicé una entrevista donde me proporcionaron especialmente las guías de uso gráfico y las reglas de negocio, tal vez esto pueda crecer hasta desarrollar una pagina completa después de ver bakend
![imagen](https://user-images.githubusercontent.com/99064239/156981144-9f5cc7d2-dbb7-4d38-a61e-36c49e83bd33.png)

![imagen](https://user-images.githubusercontent.com/99064239/156980307-3c3c53c6-59fa-4e24-995d-85d16ac612f9.png)

Me emocioné un poco con el header y que fuera responsive, ahora veo la importancía de un diseño previo UI/UX
hay ciertas nociones que puedes aplicar ya en el código pero das algunas vueltas
![imagen](https://user-images.githubusercontent.com/99064239/156981224-47d6273d-2180-4cc2-b9fc-bbfefafde1d6.png)

Para la parte del pastelero no vi muy practico llevar la parte de inventario ya que dependiendo la receta se usan diferentes ingredientes y en diferentes cantidades
![imagen](https://user-images.githubusercontent.com/99064239/156980368-9705becf-40d7-4579-89c8-cb192c460a47.png)
https://codepen.io/carlos-gmx3/pen/podYMRK
Hace tiempo ya había diseñado un menú hamburguesa y me dió mucho gusto tener la oportunidad de usarlo

**Comment by hackit92** [comment]
Quiero felicitarte no solo por cumplir de sobra con la mayoría de los requerimientos, sino por que se nota en tu post que te emociona lo que estás haciendo, que estás buscando aprender por tu parte y que estás entendiendo como aplicar esos conocimientos. Está muy padre el hecho de que tengas conocimiento del rubro, eso ayuda mucho a entender las posibles necesidades.

**Comment by JuanSalvador73** [comment]
Excelente trabajo, felicidades :D
